app/scripts/controllers/tasks.js

```diff
@@ -4,12 +4,20 @@
 /* global valueNotEmpty:false */
 
 angular.module('openhimConsoleApp')
-  .controller('TasksCtrl', function ($scope, $modal, $location, Api, Alerting, $route) {
+  .controller('TasksCtrl', function ($scope, $modal, $location, $interval, Api, Alerting, $route) {
 
     /**********************************************/
     /**         Initial load functions           **/
     /**********************************************/
 
+
+
+    var tasksInterval = $interval(function() {
+      $scope.refreshTasksList();
+    }, 5000);
```

Comment:
Will look into this. I might have to remove the loading icon then and make the status bars static and not animate